charts/linstor-cluster/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Definition of LinstorCluster
+# Ref https://github.com/piraeusdatastore/piraeus-operator/blob/v2/docs/reference/linstorcluster.md
+linstorCluster: {}
+#  apiTLS:
+#    certManager:
+#      name: linstor-cluster-api-ca
+#      kind: Issuer
+#  internalTLS:
+#    certManager:
+#      name: linstor-cluster-internal-ca
+#      kind: Issuer
+#  # name of the secret in the namespace of the installed application. Encrypting snapshot by the passphrase. Necessary for backup on S3
+#  linstorPassphraseSecret: linstor-cluster-passphrase
```

Comment:
It would be nice to have an option eg. `linstorPassphrase` which would create secret with defined value using Helm  and would not require user to perform any additional actions with other tools, eg. `kubectl`